department-of-veterans-affairs/caseflow
# Shalliburton/13404 job status

Resolves #13404 

### Description
Updates to show "Scheduling in progress" only when scheduling a virtual hearing

### Acceptance Criteria

### Testing Plan
1. Comment out line 31 in `app/models/hearings/forms/base_hearing_update_form.rb`. 1. Select judge "BVAYELLOW": http://localhost:3000/test/users
1. Navigate to hearings:  http://localhost:3000/hearings/schedule
1. Choose a "Video" hearing
1. Click the "Edit hearing details" link
1. Change from a "Video" hearing to a "Virtual" hearing
1. Once the virtual hearing has been scheduled, confirm that the hearing shows as "Scheduling in progress"
1. Uncomment the line above and then change back to "Video"
1. Once again comment the line above and change the hearing back to "Virtual"
1. Once the virtual hearing has been scheduled, confirm that the hearing shows as "Scheduling in progress"
1. After you have confirmed that toggling the hearing works, uncomment the line above to allow the hearing to finish scheduling and repeat the steps above to change the hearing back to "Virtual" only this time without commenting the line.
1. Once the hearing is set to "Virtual" comment the line above again and change the veterans email
1. Save and confirm that the virtual hearing link is disabled, but does NOT say "Scheduling in progress"
